appeals/web/src/server/appeals/appeal-details/appeal-details.mapper.js

```diff
@@ -86,6 +95,37 @@ export async function appealDetailsPage(
 		}
 	};
 
+	const caseNotes = await Promise.all(
+		appealCaseNotes.map(async (caseNote) => {
+			const createdAt = new Date(caseNote.createdAt);
+			return {
+				date: dateISOStringToDisplayDate(caseNote.createdAt),
+				day: new Intl.DateTimeFormat('en-GB', { weekday: 'long' }).format(createdAt),
+				time: dateISOStringToDisplayTime12hr(caseNote.createdAt),
+				comment: caseNote.comment,
+				user: (await mapUser(caseNote.azureAdUserId, request.session)).split('@')[0]
+			};
+		})
+	);
+
+	/** @type {PageComponent} */
+	const caseNotesComponent = {
+		type: 'details',
+		parameters: {
+			summaryText:
+				caseNotes.length === 1 ? `${caseNotes.length} case note` : `${caseNotes.length} case notes`,
```

Comment:
This could be simplified to ``${caseNotes.length} case note${caseNotes.length > 1 ? 's' : ''}``